RELEASE_NOTES.md

```diff
@@ -40,8 +50,6 @@
 - 2024-03-10 Fix paths in gen-syscall-enum tool
 - 2024-01-17 OXY-1299: implement GaugeGuard
 - 2024-01-17 OXY-1299: Implement RangeGauge metric type
-
-3.2.2
```

Comment:
It looks like you were missing tags locally to produce a proper changelog.

This will be rectified in https://github.com/cloudflare/foundations/pull/81